file_example.c

```diff
@@ -193,13 +194,26 @@ static int file_flush(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	return 0;
 }
 
+static int file_reconfig(struct tcmu_device *dev, struct tcmulib_cfg_info *cfg)
+{
+	switch (cfg->type) {
+	case TCMULIB_CFG_DEV_SIZE:
+		return tcmu_update_num_lbas(dev, cfg->data.dev_size);
```

Comment:
Yeah, I was thinking when the kernel code is merged we could merge the 2 patches in this PR to add the basic infrastructure. Then do your UA patches in another PR on top.